src/examples/features/arrays/Input.java

```diff
@@ -27,7 +27,7 @@ static public int m1(char[] c, int n) {
     for(int i =0; i < c.length; i++) {
       if(c[i] == '[') state = 1;
       else if (state == 1 & c[i] == '{') state = 2;
-      else if (state == 2 & c[i] == '<') state = 3;
+      else if (state == 2 & c[i] == '<') assert(false);
```

Comment:
This is the large example I am using on jdart-viz demo page. I changed the OK status to ERROR status because the previous doesn't have ERROR status. I just wanna make this example more useful for jdart-viz. This is minor and I have reverted this change.